conf/messages.en

```diff
@@ -1332,7 +1332,7 @@ footer.licence3=All content is available under the <a id="licence-text" href="ht
 footer.copyright=© Crown copyright
 
 #accessibility statement
-accessibility-statement.heading-and-title=Accessibility statement for Get an EORI number
+accessibility-statement.heading-and-title=Accessibility statement for Register for a customs service
```

Comment:
Don't change messages in this file.  The messages are already translated to welsh
If you need to change the text, add the same key with the new text in the -ecc version of the file.  And add the same entry to the -ecc.cy file (ideally in the same place) with the prefix "TRANSLATE"

That way it is easy to tell what needs translating.

Reply:
That's a great point. I was wondering about the messages. Thanks